### ISSUE TITLE
Dealing with quantitative indices

### ISSUE THREAD

**Issue by VincentKamphuis** [ecodistrict/IDSSDashboard#123]
Current example is the KPI Energy_label_Havensteder, calculated by the VABI policy module. The index table is the following:

1 = G, 2 = F, 3 = E, 4 = D, 5 = C, 6 = B, 7 = A, 8 = A+, 9 = A++

Can we use the G - A++ scale in the Dashboard visualisation?

**Comment by andreasrudena** [comment]
We can do something like this in a more generic way, that will say add a possibility to translate the 1-10 score with another range and call it different labels. I don't like the idea to change the dashboard for the different use cases from time to time depending on different standards. But a settings tab, something like the "context", where you set values and configuration for standards and conditions..?

**Comment by VincentKamphuis** [comment]
I agree that we should do this in a generic way. Let's discuss this on Friday.

Back-up would be to be pragmatic and make energy label a qualitative KPI.
